libultraship/libultraship/ImGuiImpl.cpp

```diff
@@ -853,9 +853,9 @@ namespace SohImGui {
 
             if (ImGui::BeginMenu("Graphics"))
             {
-                EnhancementSliderInt("Internal Resolution: %dx", "##IMul", "gInternalResolution", 1, 8, "");
+                EnhancementSliderFloat("Internal Resolution: %d %%", "##IMul", "gInternalResolution", 0.5f, 3.0f, "", 1.0f, true);
```

Comment:
```suggestion
                EnhancementSliderFloat("Internal Resolution: %d %%", "##IMul", "gInternalResolution", 0.5f, 2.0f, "", 1.0f, true);
```
Most games with an internal resolution multiplier cap at 2x. I see no reason to go higher.